### DISCUSSION TITLE
How to use form editor with antd-5 project?

### DISCUSSION THREAD

**Discussion by serhii-kulyk-dev** [alibaba/designable#388] [Q&A]
Dear developers! I was supriced you create this platform, it's very useful, you are the best. Now I need that a developer use form builder in my project and I want to use Designable as a core with antd-5 components. Can you help me use it, please?

**Comment by techwaveitsolutions** [comment]
I am also looking for the same purposes.

i am seeking for assistance, on how to use this in my existing project.

我也在寻找同样的目的。

我正在寻求帮助，了解如何在我现有的项目中使用它。
Wǒ yě zài xúnzhǎo tóngyàng de mùdì.

Wǒ zhèngzài xúnqiú bāngzhù, liǎojiě rúhé zài wǒ xiàn yǒu de xiàngmù zhōng shǐyòng tā.

**Comment by Runaho** [comment]
Hi, I was able to run the project successfully using dev container.
I created a fork you can check it out. 
[Runaho devcontainers-instructions](https://github.com/Runaho/designable/tree/devcontainers-instructions/.devcontainer)

I have created a discussion about this issue and I would appreciate it if you could take a look there.
[Discussion about dev container](https://github.com/alibaba/designable/discussions/393)
